source/dpp/runtime/app.d

```diff
@@ -277,7 +278,7 @@ private void runCPreProcessor(in string cppPath, in string tmpFileName, in strin
         auto lines = ret.
             output
             .splitLines
-            .filter!(a => !a.startsWith("#"))
```

Comment:
Maybe better to filter if it has `#` in it at all.

Reply:
It accidentally filters the lines such as `"This string contains # but it must not be filtered"`.

Reply:
Ah, I see.